apisix/plugins/dynamic-upstream.lua

```diff
@@ -187,56 +187,50 @@ function _M.check_schema(conf)
 end
 
 
-local function generate_random_num(max_val)
-    local random_val = math.random(1,max_val)
-    return random_val
-end
-
-
 local operator_funcs = {
-    ["=="] = function(v2, ctx)
-        if ctx.var[v2[1]] and ctx.var[v2[1]] == v2[3] then
+    ["=="] = function(var, ctx)
+        if ctx.var[var[1]] and ctx.var[var[1]] == var[3] then
```

Comment:
Still don't know the meaning of the variable

Reply:
example added.